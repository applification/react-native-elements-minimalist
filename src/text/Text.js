import React from 'react';
import { Text } from 'react-native';

const TextElement = props => {
  const { children } = props;

  return <Text>{children}</Text>;
};

export default TextElement;
