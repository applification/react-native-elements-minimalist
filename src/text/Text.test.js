import React from 'react';
import renderer from 'react-test-renderer';

import Text from './Text';

describe('TEXT', () => {
  it('renders text component with the text Hello', () => {
    expect(renderer.create(<Text>Hello</Text>)).toMatchSnapshot();
  });
});
