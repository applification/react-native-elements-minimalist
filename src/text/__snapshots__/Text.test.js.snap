// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TEXT renders text component with the text 'Hello' 1`] = `
<Text
  accessible={true}
  allowFontScaling={true}
  ellipsizeMode="tail"
  style={
    Object {
      "color": "blue",
    }
  }
>
  Hello
</Text>
`;

exports[`TEXT renders text component with the text Hello 1`] = `
<Text
  accessible={true}
  allowFontScaling={true}
  ellipsizeMode="tail"
  style={
    Object {
      "color": "blue",
    }
  }
>
  Hello
</Text>
`;
